### DISCUSSION TITLE
How to use translated slugs?

### DISCUSSION THREAD

**Discussion by icaliman** [camya/filament-title-with-slug#11] [Q&A]
I want to translate page titles in multiple languages with this plugin https://filamentphp.com/docs/2.x/spatie-laravel-translatable-plugin/installation

Is it possible to have translated slugs?

**Comment by camya** [comment]
Hello!

I'm not sure because unfortunately I don't have a site that requires translatable slugs. But maybe you can try it out and give me feedback. That would be great.

It is possible to use multiple instances of the "TitleWithSlugInput" component in one form, for example for multiple languages. It is also possible to use it in a repeater.
